addons/web/static/lib/jquery/jquery.js

```diff
@@ -10070,9 +10070,9 @@ jQuery.expr.pseudos.hidden = function( elem ) {
 jQuery.expr.pseudos.visible = function( elem ) {
 	return !!( elem.offsetWidth || elem.offsetHeight || elem.getClientRects().length );
 };
-
-
-
+jQuery.expr.pseudos.propValue = function (element, index, matches) {
+	return element.value === matches[3];
+},
```

Comment:
Is this a mistake? We have another file for jquery customizations (libs/jquery.js), as we typically don't patch libs in place, to prevent the customizations from being lost when updating the lib.

Reply:
The propValue selector was defined in another file (legacy/js/libs) wich was not imported in the pos. Maybe I should have done this somewhere else ?

Reply:
Why don't you import the file in the pos?

Reply:
I'll do that now, don't know why I didn't do it in the first place

Reply:
https://github.com/odoo/odoo/pull/137057
Here's the new PR @aab-odoo can you r+ ?

Reply:
thanks @robinengels